OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -33,28 +33,8937 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+                                    <msItem xml:id="ms_i1.3.6.5">
+                                        <locus target="#50ra"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe5SalPawlos">Salām-hymn to
+                                            Ṗawlos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.6.6">
+                                        <locus target="#50ra"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe5SalKirakos">Salām-hymn to
+                                            Kirākos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.6.7">
+                                        <locus target="#50ra"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe5SalHanna">Salām-hymn to
+                                            Ḥannā, the mother of Kirākos</title>
+                                    </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i1.3.7">
+                                    <locus from="50ra" to="52vb"/>
+                                    <title type="complete" ref="LIT4035SenkessarDL#Teqe6"/>
+
+                                    <msItem xml:id="ms_i1.3.7.1">
+                                        <locus from="50ra" to="50va"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Teqe6Hanna"
+                                            >Commemoration of Hannah, the mother of Samuel</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.7.2">
+                                        <locus from="50va" to="50vc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe6PantalewonSomat"
+                                            >Commemoration of ʾabbā Ṗanṭalewon</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.7.3">
+                                        <locus target="#50vc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe6Entonyos">Commemoration of
+                                            ʾƎnṭonyos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.7.4">
+                                        <locus from="50vc" to="51rc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Teqe6Enbaqom"
```

Comment:
ref "Enbaqom"  for "Habakkuk" is correct?

Reply:
The ID parts written inside ref are relevant only insofar as that they need to correspond to the ones given in the work record. They will not be printed, what will be printed is the content of the element, so here "Commemoration of Habakkuk". Here, I opted for Habakkuk because it is the Biblical prophet, but as I wrote, I am not absolutely sure of which names would be best written in English and which ones in Geez.

Reply:
I think for Non-Ethiopian saints, we should provide the name in English for a clear identification, otherwise it will be difficult to distinguish who is who.

Reply:
I'm sorry about the ref, I completely understand, but I will not correct it - it is too much work, and in this case I am very confident that they have not been used anywhere else, but as a matter of principle these IDs should only be corrected when they are wrong in a significant way.